doc/api/domain.markdown

```diff
@@ -308,56 +269,56 @@ than crashing the program.
 
 * {Array}
 
-An array of timers and event emitters that have been explicitly added
-to the domain.
+Returns an array of timers and `EventEmitter` objects that have been explicitly
```

Comment:
`domain.members` is a property, using the term `returns` suggests that it could be a function.